src/index.ts

```diff
@@ -17,7 +17,9 @@ import type { PingService } from '@libp2p/ping'
 import { KadDHT } from '@libp2p/kad-dht'
 import { PeerId } from '@libp2p/interface'
 import { filestore } from './store.js'
-import config, { configFileName } from './config.js'
+import { config, configFileName, packageJson } from './config.js'
+import { getDiskUsageStats, diskUsageCron } from './disk-usage.cron.js'
+import cors from 'cors'
```

Comment:
Install `@types/cors` to dev dependencies

Reply:
Already set line 23 package.json